supervisor/docker/manager.py

```diff
@@ -305,11 +305,13 @@ def run_command(
         stdout = kwargs.get("stdout", True)
         stderr = kwargs.get("stderr", True)
 
-        _LOGGER.info("Runing command '%s' on %s", command, image)
+        image_tag = f"{image}:{version}"
```

Comment:
Nit, makes it a bit less confusing to read.

```suggestion
        image_with_tag = f"{image}:{version}"
```